app/views/layouts/course.html.slim

```diff
@@ -3,8 +3,8 @@
   = render_breadcrumbs
   div.row
     div.col-lg-2.col-md-3.col-sm-4
-      nav role='navigation'
-        div.collapse.navbar-collapse#course-navigation-navbar
+      nav.navbar-default.sidebar role='navigation'
+        div.collapse.navbar-collapse#site-navigation-sidebar
```

Comment:
last question, why is this site navigation sidebar when it's the course template?